src/palette/view/widgets/palettewidget.cpp

```diff
@@ -1022,13 +1032,13 @@ void PaletteWidget::paintEvent(QPaintEvent* /*event*/)
         }
 
         if (idx == m_selectedIdx) {
-            c.setAlphaF(0.5);
+            c.setAlphaF(0.5f);
```

Comment:
What warning did this cause? (I'm not against, just interested)

Reply:
the compiler shows a warning about truncating the double into a float